src/AspNetCoreModuleV2/RequestHandlerLib/PipeOutputManager.cpp

```diff
@@ -2,6 +2,11 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 #include "stdafx.h"
+#include "exceptions.h"
+
+#define LOG_IF_DUPFAIL(err) do { if (err == -1) { LOG_IF_FAILED(HRESULT_FROM_WIN32(_doserrno)); }} while (0, 0);
```

Comment:
I am not a fun of macro as it breaks readability.
LOG_IF_FAILED should be DBG_LOG_IF_FAILED as it only log when debug flag enabled